### ISSUE TITLE
[Feature Request] Popper (el-popover, el-dialog, etc), option to destroy when inactive. 

### ISSUE THREAD

**Issue by nikkwong** [element-plus/element-plus#3817]
### Existing Component
Yes

### Component Name
el-popver, el-dialog

### Description
Hi, I have a page where I am using popovers in a list. The list is long; imagine maybe a row of 50-100 entries. Because el-popover only hides (does not destroy) poppers when they are inactive; they are all appended to the body of the DOM and take take up a lot of space and memory. I am wondering if we can introduce an option to destroy these poppers when they are not active. Thanks.

https://i.ibb.co/R3q9fQH/Screen-Shot-2021-10-08-at-1-40-25-PM.png

**Comment by SeminVsevolod** [comment]
I agree, for example, when using el-dropdown 50+ times, it freezes page when scrolling
